controllers/metal3machine_controller.go

```diff
@@ -257,6 +257,7 @@ func (r *Metal3MachineReconciler) reconcileNormal(ctx context.Context,
 	if bmhID == nil {
 		bmhID, err = machineMgr.GetBaremetalHostID(ctx)
 		if err != nil {
+			r.Log.Info("Failed to get the providerID for the metal3machine", "providerID", providerID)
```

Comment:
add error message and use Log.error ?

Reply:
@mboukhalfa Sorry I forgot to correspond to your comment. Updated, PTAL.
@furkatgofurov7 Thanks for your reminder.